src/index.html

```diff
@@ -8,6 +8,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="frame">
+      <div class="container">
```

Comment:
no needs to use two wrappers. It's enough to use only one (let it be container)
```suggestion
      <div class="container">
```